packages/widget-playground/src/components/DrawerControls/DesignControls/LayoutControls.tsx

```diff
@@ -0,0 +1,275 @@
+        delete defaultContainer.height;
+        delete defaultContainer.maxHeight;
+
+        setContainer(defaultContainer);
+    }
+  };
+
+  const handleInputChange: ChangeEventHandler<HTMLInputElement> = (e) => {
+    const valueConvertedToNumber = parseInt(e.target.value, 10);
+    const height = Number.isFinite(valueConvertedToNumber)
+      ? valueConvertedToNumber
+      : undefined;
+
+    setHeightValue(height);
+
+    switch (selectedLayoutId) {
+      case 'restricted-height':
+        if (getCurrentConfigTheme()?.header) {
+          setHeader();
+        }
+
+        if (height && height >= defaultMaxHeight) {
+          const containerWithMaxHeight = {
+            ...(getCurrentConfigTheme()?.container ?? {}),
+            height,
+          };
+
+          setContainer(containerWithMaxHeight);
+        }
+        break;
+      case 'restricted-max-height':
+      default:
+        if (getCurrentConfigTheme()?.header) {
+          setHeader();
+        }
+
+        if (height && height >= defaultMaxHeight) {
+          const newContainer = {
+            ...(getCurrentConfigTheme()?.container ?? {}),
+            maxHeight: height,
+          };
+
+          setContainer(newContainer);
+        }
+
+        if (!height) {
+          const newContainer = {
+            ...(getCurrentConfigTheme()?.container ?? {}),
+            maxHeight: defaultMaxHeight,
+          };
+
+          setContainer(newContainer);
+        }
+    }
+  };
+
+  return (
+    <ExpandableCard
+      title="Layout"
+      value={
+        <CardValue sx={{ textTransform: 'capitalize' }}>
+          {layoutOptions.find((option) => option.id === selectedLayoutId)
+            ?.name || ''}
+        </CardValue>
+      }
+    >
+      <ControlRowContainer
+        sx={selectedLayoutId !== 'default' ? { paddingBottom: 0 } : undefined}
+      >
+        <Select
+          value={selectedLayoutId ?? ''}
+          onChange={handleSelectChange}
+          aria-label="Layout"
+          MenuProps={{ sx: { zIndex: popperZIndex } }}
+          disabled={variant === 'drawer'}
+        >
+          {layoutOptions?.map(({ name, id }: LayoutOption) => {
+            return (
+              <MenuItem value={id} key={id}>
+                {name}
+              </MenuItem>
+            );
+          })}
+        </Select>
+      </ControlRowContainer>
+      {layoutsWithHeightControls.includes(selectedLayoutId) ? (
+        <CardRowContainer>
+          <CardRowColumn>
+            <label htmlFor="layout-height-input">
+              {inputLabel[selectedLayoutId]}
+            </label>
+            {(heightValue && heightValue < defaultMaxHeight) || !heightValue ? (
+              <CapitalizeFirstLetter variant="caption" sx={{ paddingLeft: 1 }}>
+                {`${defaultMaxHeight}px minimum`}
+              </CapitalizeFirstLetter>
+            ) : null}
+          </CardRowColumn>
+          <Input
+            id="layout-height-input"
```

Comment:
Let's add this id attribute here please to be consistent https://github.com/lifinance/widget/blob/main/packages/widget/src/utils/elements.ts#L1

Reply:
I think can remove these - I've accidently left them in from debug the routes positioning so they aren't really needed